src/frontend/Preprocessor.ml

```diff
@@ -62,6 +60,9 @@ let current_buffer () =
   let buf = Stack.top_exn include_stack in
   buf
 
+let current_location_t () =
```

Comment:
What does `_t` suffix mean?

Reply:
I honestly don't know why I put it there, removed.